runhouse/resources/hardware/cluster.py

```diff
@@ -1128,6 +1128,24 @@ def _start_or_restart_helper(
         # Update the cluster config on the cluster
         self.save_config_to_cluster()
 
+        # Save the local ~/.rh config to the cluster with the user's hashed token,
+        # if such does not exist on the cluster
+        if rns_client.token:
+            if (
+                self.run(
```

Comment:
this runs a subprocss command that will print to the console no? (also we can just use rsync i think)